presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -300,7 +301,13 @@ public SystemSessionProperties(
                         EXCHANGE_COMPRESSION,
                         "Enable compression in exchanges",
                         featuresConfig.isExchangeCompressionEnabled(),
-                        false));
+                        false),
+                booleanSessionProperty(
+                        REMOVE_DISTINCT_FROM_SEMIJOIN,
```

Comment:
Thanks for reviewing this. Yes, I agree it is not quite necessary to add a flag for this change.